ConsoleApplication1.cpp

```diff
@@ -0,0 +1,37 @@
+﻿
+//Task 1.8

```

Comment:
it's better to avoid commented code in commits

Reply:
I tried to send several tasks in one file, cause creating another one in visual studio is a complicated task. But thanks I will try to do so